packages/@react-spectrum/autocomplete/test/SearchAutocomplete.test.js

```diff
@@ -142,8 +142,8 @@ describe('SearchAutocomplete', function () {
 
   beforeAll(function () {
     user = userEvent.setup({delay: null, pointerMap});
-    jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
-    jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
+    // jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
+    // jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
```

Comment:
Removal of the clientWidth/Height mocks are illustrative only and will be reverted once I finalize this work